pkgs/development/libraries/cosmopolitan/default.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, makeWrapper, runCommand, cosmopolitan }:
+stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+  src = builtins.fetchGit {
```

```suggestion
  version = "20210228";

  src = fetchFromGitHub {
```

Please convert the fetcher to fetchFromGitHub.